src/jens/test/test_git_wrapper.py

```diff
@@ -21,135 +22,129 @@ def test_hash_object_raises_jens_git_error(self):
                           'platform-9-and-0.75')
 
     def test_gc_existing_repository(self):
-        (bare, user) = create_fake_repository(self.settings, self.sandbox_path,
+        (bare, user) = create_fake_repository(self.sandbox_path,
                                               ['qa'])
         git_wrapper.gc(bare)
 
     def test_gc_non_existing_repository(self):
         self.assertRaises(JensGitError, git_wrapper.gc, '/tmp/37d8s8dd')
 
     def test_gc_not_repository(self):
-        not_repo_path = create_folder_not_repository(self.settings,
-                                                     self.sandbox_path)
+        not_repo_path = create_folder_not_repository(self.sandbox_path)
         self.assertRaises(JensGitError, git_wrapper.gc, not_repo_path)
 
     def test_bare_clone_existing_bare_repository(self):
-        (bare, user) = create_fake_repository(self.settings, self.sandbox_path,
-                                              ['qa'])
-        git_wrapper.clone("%s/repo" % self.settings.CLONEDIR, bare, bare=True)
+        settings = Settings()
+        (bare, user) = create_fake_repository(self.sandbox_path, ['qa'])
+        git_wrapper.clone("%s/repo" % settings.CLONEDIR, bare, bare=True)
 
     def test_clone_existing_bare_repository_specific_branch(self):
-        (bare, user) = create_fake_repository(self.settings, self.sandbox_path,
+        settings = Settings()
+        (bare, user) = create_fake_repository(self.sandbox_path,
                                               ['qa', 'foo'])
-        git_wrapper.clone("%s/repo" % self.settings.CLONEDIR, bare,
+        git_wrapper.clone("%s/repo" % settings.CLONEDIR, bare,
```

Comment:
`self.settings.CLONEDIR` is okay here, I'd say. The same applies to the tests below.